src/navigation/SwipeNavigator.js

```diff
@@ -122,197 +122,205 @@ const TabBar = ({
   //     ],
   //   };
   // });
+
+  const ShadowWrapper = ({ children }) => {
```

Comment:
not sure why this file's formatting changed so much, but this `ShadowWrapper` is the only change. prevents tab bar from being unresponsivee in points tab.